src/components/DownloadLineageButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { IconButton } from '@mui/material';
+import CloudDownloadIcon from '@mui/icons-material/CloudDownload';
+import { toPng } from 'html-to-image';
+
+function downloadImage(dataUrl: string) {
+  const a = document.createElement('a');
+  a.setAttribute('download', 'lineage.png');
+  a.setAttribute('href', dataUrl);
+  a.click();
+}
+
+function DownloadButton() {
+  const download = () => {
+    toPng(document.querySelector('.react-flow') as HTMLElement, {
+      filter: (node) => {
+        console.log(node.classList)
```

Comment:
Can you remove the logging, it floods the console when exporting ;-)

Reply:
Ah yes... forgot to remove it after debugging :)